### PR TITLE
Fix Nodes form page

### PR DESCRIPTION
**Pull request by Wouter0100**
I've fixed the Nodes form page, auto completes and such.

### CODE REVIEW

File: Comments/src/View/Helper/CommentsHelper.php [hunk 1]
```diff
@@ -20,7 +22,7 @@ class CommentsHelper extends Helper {
  * beforeRender
  */
 	public function beforeRender($viewFile) {
-		if (isset($this->request->params['admin']) && !$this->request->is('ajax')) {
+		if (isset($this->request->params['prefix']) && $this->request->params['prefix'] == 'admin' && !$this->request->is('ajax')) {
```
Comment (fahad19) [inline Comments/src/View/Helper/CommentsHelper.php]: for strings, it is advised to use `===`.

File: Nodes/src/Controller/Admin/NodesController.php [hunk 2]
```diff
@@ -344,6 +345,26 @@ public function process() {
...
+	public function lookup() {
+		$this->Prg->commonProcess();
+
+		/* @var \Cake\Orm\Query $lookup */
+		$lookup = $this->Nodes->find('searchable', $this->Prg->parsedParams());
+		$lookup->contain([
+			'Users', /*'Meta', */'Taxonomies',
+		]);
+		$lookup->select([
+			'id', 'parent_id', 'type', 'user_id', 'title', 'slug',
```
Comment (fahad19) [inline Nodes/src/Controller/Admin/NodesController.php]: one problem with multiple array elements in one line, if one element is later changed, it affects the whole line along with other elements in the diff.

File: Users/src/Controller/Admin/UsersController.php [hunk 3]
```diff
@@ -5,6 +5,7 @@
 use Cake\Cache\Cache;
 use Cake\Core\Configure;
 use Cake\Network\Email\Email;
+use Cake\ORM\Query;
```
Comment (fahad19) [inline Users/src/Controller/Admin/UsersController.php]: didn't know that Croogo uses Cake's new ORM now. nice!

File: Comments/src/View/Helper/CommentsHelper.php (hunk 1 above)
Comment (Wouter0100) [inline Comments/src/View/Helper/CommentsHelper.php]: Hmm, okay. Thx! Will fix that :+1:

File: Nodes/src/Controller/Admin/NodesController.php (hunk 2 above)
Comment (Wouter0100) [inline Nodes/src/Controller/Admin/NodesController.php]: True story. Will update that later :smile:

File: Users/src/Controller/Admin/UsersController.php (hunk 3 above)
Comment (Marlinc) [inline Users/src/Controller/Admin/UsersController.php]: :+1:

**Comment by Wouter0100** [comment]
Rebased against 3.0

**Pull request merged**